scenarios/single_write_many_read/src/main.rs

```diff
@@ -9,14 +9,12 @@ struct ScenarioValues {
 
 impl UserValuesConstraint for ScenarioValues {}
 
-fn setup(ctx: &mut RunnerContext<HolochainRunnerContext>) -> HookResult {
-    configure_app_ws_url(ctx)?;
-    Ok(())
-}
-
 fn agent_setup(
     ctx: &mut AgentContext<HolochainRunnerContext, HolochainAgentContext<ScenarioValues>>,
 ) -> HookResult {
+    run_holochain_conductor(ctx)?;
```

Comment:
this pattern is repeated in every binary. Do you think it could make sense to create a helper wrapping it?